app/main.py

```diff
@@ -1,3 +1,27 @@
+from app.knights import Knight
+from app.fight import Arena
+
+
+def battle(knights: dict) -> dict:
```

Comment:
It is not a good idea to name two functions of method in the same name in such small project. I would prefer to rename one of those

Reply:
You can create objects of knight here directly from the dict and put them as arguments to any functions and methods you need